src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -148,6 +176,10 @@ public static PhysicalAddress Parse(string address)
                 {
                     value -= 0x37;
                 }
+                else if (value >= 0x61 && value <= 0x66) // lowercase
```

Comment:
Honestly, just because the other conditionals were doing the same type of comparison to hex values (see lines 171, 175). I'm not a fan of the hex comparisons because it's harder to read (hence the //lowercase comment), but I figured it would be better to remain consistent with the previous comparisons. Though, I suppose the delimiters are compared via (int) cast. 

I can change this, but I'd also want to change those lines (171, 175) as well to keep it consistent.